src/BenchmarkDotNet/Detectors/Cpu/Linux/LinuxCpuInfoParser.cs

```diff
@@ -28,12 +31,13 @@ private static class Lscpu
 
     /// <param name="cpuInfo">Output of `cat /proc/cpuinfo`</param>
     /// <param name="lscpu">Output of `lscpu`</param>
-    internal static CpuInfo Parse(string? cpuInfo, string? lscpu)
+    internal static CpuInfo Parse(string cpuInfo, string lscpu)
```

Comment:
This change appears to be incorrect. It is valid for one of these arguments to be null while the other is not. In `LinuxCpuDetector`, we only check the case when both values are null. As a result, we could end up with a null value in a non-nullable string.